docs/infrastructure/madison-itb.md

```diff
@@ -0,0 +1,304 @@
+3. `scp -p` the certificate(s) (`*cert.pem*` and `*key.pem`) to your home directory on `osghost` or whatever machine you use for Ansible
+4. Find the corresponding certificate location(s) in the Ansible `roles/certs/files` directory
+5. `cp -p` the certificate files over the top of the existing Ansible ones (or create new, equivalent paths)
+6. Run `ansible-vault encrypt FILE(S)` to encrypt the files — get the Ansible vault password from Tim C. if you need it
+7. Verify permissions, contents (you can `cat` the encrypted files), etc.
+8. Apply the files with something like `ansible-playbook secure.yml -i inventory -bK -f 20 -t certs`
+9. Commit changes (now or after applying)
+10. Push changes to origin
+
+#### Doing yum updates
+
+1. Check to see if updates are needed and, if so, what would be updated:
+
+        :::console
+        ansible [ HOST | GROUP ] -i inventory -bK -f 20 -m command -a 'yum check-update'
+
+    You can name a single `HOST` or an inventory `GROUP` (such as the handy `current` group); with a group, you can
+    further restrict the hosts with a `-l` option.
+
+    **Note:** `yum check-update` exits with status code `100` when it succeeds in identifying packages to update;
+    therefore Ansible shows such results as failures.
+
+2. Review the package lists to be updated and decide whether to proceed with all updates or limited ones
+
+3. Do updates:
+
+        :::console
+        ansible [ HOST | GROUP ] -i inventory -bK -f 20 -m command -a 'yum --assumeyes update' [ -l LIMITS ]
+
+#### Updating HTCondor from Upcoming
+
+Something like this:
+
+    :::console
+    ansible condordev -i inventory -bK -f 10 -m command -a 'yum --enablerepo=osg-upcoming --assumeyes update condor'
+
+
+## Monitoring
+
+### HTCondor-CE View
+
+Once we sort out our firewall rules, pilot, VO, and schedd availability graphs should be available
+[here](http://itb-ce1.chtc.wisc.edu:59619) through HTCondor-CE View.
+
+### Tracking payload jobs via Kibana
+
+At this time, the easest way to verify that payload jobs are running within the glideinWMS pilots is to track their records via <literal><a href="<https://gracc.opensciencegrid.org/kibana/app/kibana#/discover?_g>=(refreshInterval:(display:Off,pause:f,value:0),time:(from:now%2Fw,mode:quick,to:now%2Fw))&\_a=(columns:!(\_source),index:%27gracc.osg-itb.raw-\*%27,interval:auto,query:(query\_string:(analyze\_wildcard:!f,query:%27\*%27)),sort:!(EndTime,desc))">Kibana</a></literal>. To view all payload jobs that have run on our ITB site in the past week, use <literal><a href="<https://gracc.opensciencegrid.org/kibana/app/kibana#/discover?_g>=(refreshInterval:(display:Off,pause:f,value:0),time:(from:now%2Fw,mode:quick,to:now%2Fw))&\_a=(columns:!(ProbeName),index:%27gracc.osg-itb.raw-\*%27,interval:auto,query:(query\_string:(analyze\_wildcard:!f,query:%27ResourceType:%20Payload%20AND%20ProbeConfig:%20%22condor:itb-ce1.chtc.wisc.edu%22%27)),sort:!(EndTime,desc))">this query</a></literal>.
```

Comment:
These links are real broken. I tried to use the URL shortener in our Kibana but it seems to be broken. I'm awaiting the GRACC team's response.